core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.spec.ts

```diff
@@ -183,6 +185,44 @@ describe('Translation language selector', () => {
     expect(component.activeLanguageCode).toBe('en');
   });
 
+  it('should display the preferred language when the preferred' +
+    ' language is defined', async(() => {
+    preferredLanguageCode = 'en';
+    let languageDescription = component.languageIdToDescription.en;
```

Comment:
`constants.SUPPORTED_AUDIO_LANGUAGES.en` is not defined since we are not having key-value pairs in `SUPPORTED_AUDIO_LANGUAGES` so I have used `find()` method in order to deal with this.